tests/unit/accounts/test_urls.py

```diff
@@ -0,0 +1,32 @@
+from django.urls import reverse_lazy
+from rest_framework import status
+from rest_framework.test import APITestCase, APIClient
+from allauth.account.models import EmailAddress
+from django.contrib.auth.models import User
+import unittest
```

Comment:
Also, please check the PEP8 styling for imports here https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style